test/sql_tracker_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+module SqlTracker
+  class HandlerTest < Minitest::Test
+    def test_tracking_queries_with_a_block
+      config = SqlTracker::Config.apply_defaults
+      config.enabled = true
+      config.tracked_sql_command = %w(SELECT INSERT)
+
+      expected_queries = [
+        'SELECT * FROM users',
+        'insert into users VALUES (xxx)'
+      ]
+
+      query_data = SqlTracker.track do
+        expected_queries.each { |q| instrument_query(q) }
+        instrument_query('DELETE FROM users WHERE id = 1')
+      end
+
+      instrument_query('SELECT * FROM comments')
+
+      assert_equal(
```

Comment:
I think the first argument should be expected_queries in `assert_equal`

Reply:
Fixed